### PR TITLE
Feed experiment for 08-23-2022

### PR DESCRIPTION
**Pull request by joshpuetz**
<!--
     For Work In Progress Pull Requests, please use the Draft PR feature,
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

Feed experiment for 08-23-2022 as outlined in the [feed experiments docs](https://docs.google.com/document/d/1H1WsktwvlspsB_1RmsZxfVcJHVcaeorHLjpAWG8Oazw/edit?usp=sharing). Notably this experiment takes the incumbent from 07-26-2022, and swaps it's `public_reactions` lever for the the `public_reactions_scores` lever. Weight is 80% incumbent, 20% new challenger.

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: covered by existing tests
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: config/field_test.yml [hunk 1]
```diff
@@ -28,10 +28,35 @@
 ################################################################################
 experiments:
   # NOTE: Our feed strategy testing experiment must begin with "feed_strategy"
-  feed_strategy_starting_20220801:
+  feed_strategy_starting_20220823:
     # NOTE: Required as we want only want to consider for conversion events that
     #       occurred on or after the given start_date.
     started_at: 2022-08-01
+    ended_at: 2022-08-23
```
Comment (maestromac) [inline config/field_test.yml:35]: shouldn't there just be the start time only? because it will be the new current test?

Comment (joshpuetz) [reply config/field_test.yml:35]: Aaah, thought I got all the copy pasta. Thanks!

File: config/field_test.yml [hunk 2]
```diff
@@ -28,10 +28,34 @@
 ################################################################################
 experiments:
   # NOTE: Our feed strategy testing experiment must begin with "feed_strategy"
-  feed_strategy_starting_20220801:
+  feed_strategy_starting_20220823:
     # NOTE: Required as we want only want to consider for conversion events that
     #       occurred on or after the given start_date.
     started_at: 2022-08-01
```
Comment (maestromac) [inline config/field_test.yml:34]: wrong start time? :laughing: 
```suggestion
    started_at: 2022-08-03
```

Comment (joshpuetz) [reply config/field_test.yml:34]: I swear I actually edited this file correctly! Somehow I buggered it up!

**Review by maestromac — APPROVED** [review]

**Review by Ridhwana — APPROVED** [review]

**Pull request merged**